ringpop.go

```diff
@@ -286,6 +286,18 @@ func (rp *Ringpop) Bootstrap(opts *swim.BootstrapOptions) ([]string, error) {
 		}
 	}
 
+	identity, err := rp.identity()
+	if err != nil {
+		return nil, err
+	}
+
+	// Check we're in the bootstrap host list and add ourselves if we're not
+	// there. If the host list is empty, this will create a single-node
+	// cluster.
+	if !stringInSlice(opts.Hosts, identity) {
+		opts.Hosts = append(opts.Hosts, identity)
```

Comment:
this'll modify the opts.Hosts passed in by the caller.. which is not desirable. I'd recommend a copy of the hosts.